content/assignments/techjobs-console/_index.md

```diff
@@ -0,0 +1,477 @@
+[overloading](https://www.geeksforgeeks.org/c-sharp-method-overloading/).
+
+Here are some questions to ask yourself while reading this code:
+
+1. What is the data type of a "job" record?
+1. Why does `FindAll(string)` return something of type `List<string>`
+   while `FindByColumnAndValue(string, string)` and `FindAll()` return
+   something of type `List<Dictionary<string, string>>`?
+1. Why is `LoadData()` called at the top of each of these four methods? Does
+   this mean that we load the data from the CSV file each time one of them
+   is called?
+
+## Your Tasks
+
+<!-- TODO: Add reference back to Assignment 0 -->
+
+Before diving into your tasks, review Assignment 0 for details on running the autograding tests for this assignment. This assignment has multiple tests, and we highly recommend the following workflow:
+
+1. Write the code for the task, verifying manually that it works by running the `TechJobsConsoleAutograded` project.
+1. When you think you've completed a task, run the individual test that corresponds to the task. 
+1. If the test fails, review the test output and go back to your code to try to fix it.
+1. Once the single test passes, run *all* of the tests to make sure you didn't break any tests that previously passed.
+1. Repeat this process until all tests pass. 
+
+Now we'll outline the tasks for your first apprenticeship assignment.
+
+### Implement `PrintJobs()`
+
+When trying out the program, and later when reading the code, you
+hopefully noticed that there's some work to do in the `PrintJobs()`
+method. As it stands, it currently just prints a message:
+`"PrintJobs is not implemented yet"`.
+
+Complete this method. It should print out jobs *in this precise format*:
+
+```bash
+   *****
+   position type: Data Scientist / Business Intelligence
+   name: Sr. IT Analyst (Data/BI)
+   employer: Bull Moose Industries
+   location: Saint Louis
+   core competency: Statistical Analysis
+   *****
+
+   *****
+   position type: Web - Back End
+   name: Ruby specialist
+   employer: LaunchCode
+   location: Saint Louis
+   core competency: Javascript
+   *****
+```
+
+For the autograding script to correctly grade your code, you'll need to match this format *exactly*. In particular, note the number of asterisks surrounding each listing, and the blank line between listings.
+
+If there are no results, it should print `No results`. Again, you should use this *exact* message.
+
+{{% notice orange "Warning" "rocket" %}}
+
+   To create new lines for your output, use `Environment.NewLine`.  
+
+{{% /notice %}}
+
+Using the `Environment.NewLine`  will allow the autograding unit tests pass regardless of your operating system.  `\n` is a new line in Mac OS, but will be read as `\r\n` in Windows.    Read about the differences in line breaks [here](https://dev.to/pieter/why-windows-uses-rn-newlines-instead-of-n-126l).  Read more about how `Environment.NewLine` [works](https://www.dotnetperls.com/newline).
```

Comment:
This is making it more complicated than it should be, best practices is to use Environment.Newline

```suggestion
Traditionally `\n` is a new line in Mac OS and `\r\n` is new line in Windows. `Environment.NewLine`  is the universal way to create a new line and works regardless of your operating system.  Allowing code written on a Mac to pass unit tests when the same code is run on Windows.
```